test/unit/color/p5.Color.js

```diff
@@ -781,7 +781,7 @@ suite('p5.Color', function() {
     });
   });
 
-  suite.only('p5.Color.prototype.toString', function() {
+  suite('p5.Color.prototype.toString', function() {
```

Comment:
Oh, I was wrong about this not affecting tests from the command-line.  It *does* affect tests from the command-line!  In fact, f024aab406eafbc58bb1ea0a0d72936cb0a62f7e broke the tests, but they didn't show up in Travis CI because of this line :disappointed: